app/ui/content/views/stars/star-item.jsx

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright 2016 Mozilla
+
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ this file except in compliance with the License. You may obtain a copy of the
+ License at http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software distributed
+ under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+
+import Style from '../../../shared/style';
+
+import * as ContentPropTypes from '../../model/content-prop-types';
+import ListItem from '../../../shared/widgets/list-item';
+
+const STAR_ITEM_STYLE = Style.registerStyle({
```

Comment:
This is duplicated between stars and history view.  I'd suggest move it to the shared ListItem component in LIST_ITEM_STYLE, until we have a third consumer that doesn't need this style at which point we could break it out.